src/common/runtime-key.js

```diff
@@ -0,0 +1,12 @@
+define([
+	"../util/string/hash"
+], function( stringHash ) {
+
+return function( fnName, locale, args ) {
+	var hash;
+	args = [].slice.call( args, 0 );
```

Comment:
At some point of the implementation, formatters/parsers `arguments` as is were been passed. But, now an Array clone is always the case. So, this isn't needed anymore indeed. Thanks